airflow/example_dags/docker_copy_data.py

```diff
@@ -0,0 +1,84 @@
+'''
+This sample "listen to directory". move the new file and print it, using docker-containers.
+The following operators are being used: DockerOperator, BashOperator & ShortCircuitOperator.
+TODO: Review the workflow, change it accordingly to to your environment & enable the code.
+'''
+
+# from __future__ import print_function
```

Comment:
p.s. you have another code example: https://github.com/asnir/airflow/commit/f1063c9dfadd65a4a1b00398f40245679185e4c2
```python
t3 = DockerOperator(api_version='1.19',
     docker_url='tcp://localhost:2375',
     command='/bin/sleep 30',
     image='centos:latest',
     network_mode='bridge',
     task_id='docker_op_tester',
     dag=dag)
 
```